src/content/en/fundamentals/performance/http2/index.md

```diff
@@ -0,0 +1,236 @@
+project_path: /web/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+description: HTTP/2 will make our applications faster, simpler, and more robust
```

Comment:
this is the meta description, mostly meant for search, I'd recommend something more along the lines of what people would search for.